src/index.js

```diff
@@ -17,7 +17,8 @@ exports.addPrefix = (multicodecStrOrCode, data) => {
       throw new Error('multicodec not recognized')
     }
   }
-  return Buffer.concat([pfx, data])
+  let encodedPrefix = new Buffer(varint.encode(bufferToNumber(pfx)))
```

Comment:
We don't need to calculate these every single time, the `base-table` could have the varint buffers all calculated when the module is required